test/e2e/diff_test.go

```diff
@@ -120,8 +120,9 @@ data:
 	})
 
 	logger.Section("deploy no change", func() {
-		out, _ := kapp.RunWithOpts([]string{"deploy", "-f", "-", "-a", name, "--json"},
+		out, err := kapp.RunWithOpts([]string{"deploy", "-f", "-", "-a", name, "--json"},
 			RunOpts{IntoNs: true, StdinReader: strings.NewReader(yaml1)})
+		require.NoError(t, err)
```

Comment:
no need to check on err -- by default kapp object should do it for you unless AllowError is set to true. (this just makes it less noisy to write tests)

Reply:
I'll do same error handling in RunEmbedded then

Reply:
Done